crates/graphql/src/schema.rs

```diff
@@ -941,7 +959,7 @@ impl QueryRoot {
             .load(&conn)
             .unwrap();
 
-        rows.into_iter().map(Into::into).collect()
+        Ok(rows.into_iter().map(Into::into).collect())
```

Comment:
This function calls `.unwrap()` in several spots, so if the return type is to be changed to Result the panics should also be removed.

Reply:
yeah this needs to be changed in whole schema.rs

Reply:
Right.  For now I'd suggest just removing the FieldResult if possible and unwrapping.

Reply:
I have changed FieldResult to Result<Vec<Nft>,FieldError>

Reply:
I think having FieldResult was fine, as long as all unwraps are changed to the `?` operator.